docs/how_to_guides/pushing_and_pulling.mdx

```diff
@@ -140,7 +140,7 @@ oras pull localhost:5000/hello-artifact:v2
 
 ## Using cache when pulling artifacts
 
-In order to save unnecessary network bandwidth and disk I/O oras should provides a solution to pull the artifacts into a local content-address storage (CAS) if the content does not exist, and then copy the artifact to the desired storage.
+In order to save unnecessary network bandwidth and disk I/O, oras provides a solution to pull the artifacts into a local content-address storage (CAS) if the content does not exist, and then copy the artifact to the desired storage.
 
```

Comment:
```suggestion
Oras pulls the artifact into a content-addressable storage (CAS) cache if the content is not available locally to save bandwidth and disk I/O.
Once the content is availabe in the cache, oras copies the artifact to the desired location. 
```

@Wwwsylvia does the able statement sound accurage. My hope is to split up the long sentence and call out that the cache is a CAS.

Reply:
thank you @sajayantony for helping

Reply:
LGTM